tests/test_inverters.py

```diff
@@ -0,0 +1,19 @@
+""" Test for main app """
+
+from pv_site_api.pydantic_models import Inverters
+
+
+def test_get_inverters_fake(client, fake):
+    response = client.get("/inverters")
+    assert response.status_code == 200
+
+    inverters = Inverters(**response.json())
+    assert len(inverters.inverters) > 0
+
+
+# def test_get_inverters(db_session, client, forecast_values):
+#     response = client.get(f"/sites/{site_uuid}/clearsky_estimate")
+#     assert response.status_code == 200
+
+#     clearsky_estimate = ClearskyEstimate(**response.json())
+#     assert len(clearsky_estimate.clearsky_estimate) > 0
```

Comment:
Next, we'll want to uncomment this test (and change the endpoint in it). Here's the issue: your endpoints will end up trying to make a request to Enode, but we shouldn't do that in the test. Instead, you'll use `https://pypi.org/project/pytest-httpx/` to have the Enode API requests get intercepted and swapped with fake data. Read the docs there for info, then add the library via `poetry add -D pytest-httpx`. You'll first need to bump the version of `httpx` in the `pyproject.toml` to `0.24.0`.

Then, add this code to `conftest.py`:
```python
@pytest.fixture
def non_mocked_hosts() -> list:
    return ["testserver"]
```

Then, in your tests, add `httpx_mock` as a parameter (to the non fake ones) and use it to mock a Enode request:
```python
httpx_mock.add_response(url="https://enode-api.production.enode.io/inverters")
```